chapter3/section3.x/ActualMeterReads.md

```diff
@@ -0,0 +1,14 @@
+## ActualMeterReads request
```

Comment:
chapter3/section3.x is not correct
the smartmetering documentation is located in chapter3/section5
I think chapter3/smartmetering/ is a better choice